data/2022-results.json

```
@@ -1,126 +1,126 @@
 {
-  "survey-start-date": "2022-06-03",
-  "survey-end-date": "2022-06-17",
-  "total-number-responses": 123,
-  "total-trip-count": 100,
-  "total-distance": 12300,
-  "total-co2-emissions-tonnes": 8888,
+  "survey-start-date": "2022-07-12",
+  "survey-end-date": "2022-07-27",
+  "total-number-responses": 162,
+  "total-trip-count": 1068,
+  "total-distance": 13921,
+  "total-co2-emissions-tonnes": 2.91,
   "work-mode": {
-    "onsite": 5,
-    "hybrid": 50,
-    "wfh": 20
+    "onsite": 66,
+    "hybrid": 86,
+    "wfh": 10
   },
   "commute-days-distribution": {
-    "Mon": 15,
-    "Tue": 10,
-    "Wed": 25,
-    "Thu": 20,
-    "Fri": 30,
-    "Sat": 0,
+    "Mon": 17.98,
+    "Tue": 22.28,
+    "Wed": 22.28,
+    "Thu": 21.16,
+    "Fri": 16.10,
+    "Sat": 0.19,
     "Sun": 0
   },
   "TopThreeData": {
     "Distance": [
       {
-        "name": "Train",
-        "count": 652
+        "name": "Car",
+        "count": 11651
       },
       {
-        "name": "Car",
-        "count": 254
+        "name": "Train/tram",
+        "count": 1020
       },
       {
-        "name": "Walk",
-        "count": 150
+        "name": "Bicycle",
+        "count": 444
       }
     ],
     "Trip Count": [
       {
-        "name": "Train",
-        "count": 350
+        "name": "Car",
+        "count": 796
       },
       {
-        "name": "Car",
-        "count": 562
+        "name": "Walk/run",
+        "count": 106
       },
       {
-        "name": "Walk",
-        "count": 113
+        "name": "Bicycle",
+        "count": 64
       }
     ],
     "Emission": [
       {
-        "name": "Bus",
-        "count": 956
+        "name": "Car",
+        "count": 2.843
```

let's round this result to 2 significant decimals places
```suggestion
        "count": 2.84
```

---

then we need to round Carpool and train , which means they will be equal 

```json
"Emission": [
      {
        "name": "Car",
        "count": 2.843
      },
      {
        "name": "Carpool",
        "count": 0.034
      },
      {
        "name": "Train/tram",
        "count": 0.030
      }
    ]
```

will be:

```json
"Emission": [
      {
        "name": "Car",
        "count": 2.84
      },
      {
        "name": "Carpool",
        "count": 0.03
      },
      {
        "name": "Train/tram",
        "count": 0.03
      }
    ]
```

---

If you are happy with that, I'll round them. Alternatively we can convert it back into kg and it will be:

```json
"Emission": [
      {
        "name": "Car",
        "count": 2843
      },
      {
        "name": "Carpool",
        "count": 34
      },
      {
        "name": "Train/tram",
        "count": 30
      }
    ]
```